plugins/org.eclipse.elk.core/src/org/eclipse/elk/core/LayoutConfigurator.java

```diff
@@ -218,16 +219,20 @@ public void visit(final ElkGraphElement element) {
     @SuppressWarnings("unchecked")
     protected void applyProperties(final ElkGraphElement element, final IPropertyHolder properties) {
         if (properties != null) {
-            if (!optionFilters.isEmpty()) {
-                for (Map.Entry<IProperty<?>, Object> entry : properties.getAllProperties().entrySet()) {
-                    boolean accept = optionFilters.stream()
-                            .allMatch(filter -> filter.accept(element, entry.getKey()));
-                    if (accept) {
-                        element.setProperty((IProperty<Object>) entry.getKey(), entry.getValue());
+            List<IOptionFilter> filters = getFilters();
```

Comment:
Is there a reason to use `getFilters()` instead of the previous `optionFilters`?

Reply:
The method is not final, so theoretically one could override that.